### PR TITLE
Added a feature to close a report

### PR DESCRIPTION
**Pull request by flashburst**
- The `closeReport` feature allows Governance Admin to emergency resolve an incident when the incident which is reported is a **False Reporting**. This will help to reduce the spam incident reports in the protocol. #117

### CODE REVIEW

File: contracts/core/governance/resolution/Resolvable.sol [hunk 1]
```diff
@@ -178,6 +178,48 @@ abstract contract Resolvable is Finalization, IResolvable {
...
+
+    // solhint-disable-next-line not-rely-on-time
+    s.setUintByKeys(ProtoUtilV1.NS_GOVERNANCE_RESOLUTION_TS, coverKey, productKey, block.timestamp);
+
+    // solhint-disable-next-line not-rely-on-time
+    s.setUintByKeys(ProtoUtilV1.NS_RESOLUTION_DEADLINE, coverKey, productKey, block.timestamp);
+
+    _resolve(coverKey, productKey, incidentDate, false, true);
+    _finalize(coverKey, productKey, incidentDate);
+  }
```
Comment (binodnpm) [inline contracts/core/governance/resolution/Resolvable.sol:221]: Let us also emit an event:

```solidity
event ReportClosed(bytes32 indexed coverKey, bytes32 indexed productKey, address indexed closedBy, uint256 incidentDate);
```

Comment (flashburst) [reply contracts/core/governance/resolution/Resolvable.sol:223]: Added `ReportClosed` event

File: contracts/core/governance/resolution/Resolvable.sol [hunk 2]
```diff
@@ -178,6 +178,50 @@ abstract contract Resolvable is Finalization, IResolvable {
...
+  ) external override nonReentrant {
+    require(incidentDate > 0, "Please specify incident date");
+
+    s.mustNotBePaused();
+    AccessControlLibV1.mustBeGovernanceAdmin(s);
+
+    s.mustBeSupportedProductOrEmpty(coverKey, productKey);
+    s.mustBeValidIncidentDate(coverKey, productKey, incidentDate);
+    s.mustBeDuringReportingPeriod(coverKey, productKey);
+    s.mustBeDisputed(coverKey, productKey);
```
Comment (rudolfnep) [inline contracts/core/governance/resolution/Resolvable.sol:210]: No need to consume additional gas by having a requirement to first dispute the report before closing it.

Comment (binodnpm) [inline contracts/core/governance/resolution/Resolvable.sol:210]: No need to consume additional gas by having a requirement to first dispute the report before closing it.

Comment (flashburst) [reply contracts/core/governance/resolution/Resolvable.sol]: Removed it

**Comment by binodnpm** [comment]
Please target the [rc2/audit-complete](https://github.com/neptune-mutual-blue/protocol/tree/rc2/audit-complete) branch

**Review by binodnpm — APPROVED** [review]

**Pull request merged**